admin/scripts/zvlastni/reporty/stravenky.xtpl

```diff
@@ -1,20 +1,27 @@
 <!-- begin:stravenky -->
 <style>
 
-.radek { border: solid 0.3mm; page-break-inside: avoid; display: inline-block; font-family: sans-serif; border-left: none; }
-.radek > div { float: left; width: 16mm; height: 16mm; padding: 2mm; border-left: solid 0.3mm; overflow: hidden; position: relative; }
+.radek { page-break-inside: avoid; display: inline-block; font-family: sans-serif; border-left: none; }
+.radek > div { float: left; width: 22mm; height: 22mm; padding: 2mm; box-shadow:
+    1px 0 0 0,
+    0 1px 0 0,
+    1px 1px 0 0,   /* Just to fix the corner */
+    1px 0 0 0 inset,
+    0 1px 0 0 inset; overflow: hidden; position: relative; }
 .radek > hr { margin: 0; border: 0; clear: both; height: 0; }
-.jidlo > img { width: 100%; position: absolute; top: 0; left: 0; opacity: 0.5; }
+.jidlo { }
+.jidlo > img{ height: 100%; position: absolute; top: 0; right: 0; opacity: 0.5; }
 
 </style>
 
 <!-- begin:uzivatel -->
 <div class="radek">
-  <div>{login_uzivatele} ({id_uzivatele})</div>
+  <div style="font-weight: bold;">{login_uzivatele} ({id_uzivatele})</div>
   <!-- begin:jidlo -->
   <div class="jidlo">
     <img src="../files/design/logo.png">
-    <div style="position:absolute">{nazev}</div>
+    <div style="font-weight: bold;">{nazev}</div>
+    <div style="color: #444; font-style: italic; text-align: right; font-size: 11pt; margin-top: 2mm;">{login_uzivatele} ({id_uzivatele})</div>
```

Comment:
Tak dlouhý styl už by se měl vytvořit jako samostatná css třída. Jednotky by měly být mm nebo %, pokud všude jinde je mm.